src/test/eventSchemas/explorer/registryItemNode.test.ts

```diff
@@ -16,13 +16,24 @@ import {
     assertNodeListOnlyContainsErrorNode,
     assertNodeListOnlyContainsPlaceholderNode,
 } from '../../utilities/explorerNodeAssertions'
-import { MockSchemaClient, MockToolkitClientBuilder } from '../../shared/clients/mockClients'
 import { clearTestIconPaths, IconPath, setupTestIconPaths } from '../../shared/utilities/iconPathUtils'
 import { asyncGenerator } from '../../utilities/collectionUtils'
 import globals from '../../../shared/extensionGlobals'
+import * as sinon from 'sinon'
+
+function createSchemaClient(data?: { schemas?: Schemas.SchemaSummary[]; registries?: Schemas.RegistrySummary[] }) {
+    return {
+        regionCode: 'code',
+        async *listSchemas(registryName: string, version: string): AsyncIterableIterator<Schemas.SchemaSummary> {
+            yield* data?.schemas ?? []
+        },
+        async *listRegistries() {
+            yield* data?.registries ?? []
+        },
+    } as unknown as SchemaClient
```

Comment:
Although we might plan to improve the patterns and practices for testing SDKs / service clients in the future, we can start by mentioning this pattern in our guidelines somewhere (also point to an existing concrete test).

Reply:
If we want to use this pattern, I can write up a factory function real quick that does this with proper types.

Reply:
I'll follow up with something more opinionated plus docs in another PR.
Should be able to write a nice, simple abstraction that uses `sinon` but with the class-based ergonomics of `mockito`.